### PR TITLE
Improve UX of (what will likely be) landing page

### PR DESCRIPTION
**Pull request by kjdchapman**
Main thing is changes to copy.

Also added a password reset link and more hints (for users who aren't organisations, and around which emails are valid)

image

### CODE REVIEW

File: app/views/users/registrations/new.html.erb [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
-    <p class="govuk-body">
-      We are glad you would like to sign up your organisation for GovWifi.
-    </p>
-    <p class="govuk-body">
-      You have the option to sign up for a three month trial. After you have
-      tested GovWifi, you can go live and scale it at your organisation for free!
+    <h1 class="govuk-heading-l">Sign up your Organisation to GovWifi</h1>
+
+    <p class="govuk-!-padding-bottom-1">
+      <%= link_to "If you are trying to connect to GovWifi as an end user, see instructions here", "", class: "govuk-link govuk-!-font-size-19" %>
```
Comment (tommotaylor) [inline app/views/users/registrations/new.html.erb]: I would suggest just making the end bit the link. More Govuk'y. Eg:
screen shot 2018-08-23 at 11 52 58

File: app/views/users/registrations/new.html.erb [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
+          <div class="govuk-hint"> Must be a valid government email address </div>
           <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "govuk-input", required: true %>
         </div>
-
         <div class="actions">
           <%= f.submit "Sign up", class: "govuk-button govuk-!-margin-top-4" %>
         </div>
       <% end %>
 
+      <%= link_to "Forgot password", new_password_path(resource_name), class: "govuk-link govuk-!-font-size-19" %><br />
```
Comment (tommotaylor) [inline app/views/users/registrations/new.html.erb]: This already exists and can be displayed dynamically from the `users/shared/links` partial.

File: app/views/users/registrations/new.html.erb (hunk 1 above)
Comment (kjdchapman) [reply app/views/users/registrations/new.html.erb]: Good shout:

image

Commit coming shortly..

File: app/views/users/registrations/new.html.erb (hunk 2 above)
Comment (kjdchapman) [reply app/views/users/registrations/new.html.erb]: Yeah, I looked in the partial, saw `controller_name !=` and ran away. 

I'm worried about how much time I have for sorting out these pages today, but I can try and fix this if you think it's best?

Comment (kjdchapman) [reply app/views/users/registrations/new.html.erb]: Talked about this IRL and Tom pointed out the very easy solution - nice one Tom!

**Pull request merged**